core/server/data/schema/schema.js

```diff
@@ -52,90 +52,90 @@ module.exports = {
         updated_by: {type: 'string', nullable: true}
     },
     roles: {
-        id: {type: 'string', nullable: false, primary: true},
-        name: {type: 'string', maxlength: 150, nullable: false, unique: true},
-        description: {type: 'string', maxlength: 200, nullable: true},
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
+        name: {type: 'string', maxlength: 50, nullable: false, unique: true},
+        description: {type: 'string', maxlength: 2000, nullable: true},
         created_at: {type: 'dateTime',  nullable: false},
         created_by: {type: 'string',  nullable: false},
         updated_at: {type: 'dateTime',  nullable: true},
         updated_by: {type: 'string',  nullable: true}
     },
     roles_users: {
-        id: {type: 'string', nullable: false, primary: true},
+        id: {type: 'string', maxlength: 24, nullable: false, primary: true},
         role_id: {type: 'string', nullable: false},
         user_id: {type: 'string', nullable: false}
```

Comment:
I can see some field which don't have the 24 length definition.
e.g. created_by, user_id